src/parser.rs

```diff
@@ -156,75 +159,72 @@ pub fn parse_catalog<'a, R: io::Read>(mut file: R, opts: ParseOptions) -> Result
     }
 
     let mut catalog = Catalog::new();
-    let resolver = match opts.force_plural {
-        Some(func) => Some(Resolver::Function(func)),
-        None => None,
-    };
+    let mut resolver = opts.force_plural.map(Resolver::Function);
```

Comment:
This could probably be moved to line 223, to have all logic of resolver in one place, i.e.:

    let resolver = opts.force_plural.map(Resolver::Function).unwrap_or(/* parse AST or use default `n != 1` */)

Reply:
Well, I reviewed commit-by-commit and this was later removed :smile: But I hope we reinstate the `Function` resolver and this comment will be relevant again.